images/win/scripts/Tests/PHP.Tests.ps1

```diff
@@ -0,0 +1,12 @@
+Describe "PHP" {
+    It "Check <ToolName> in the PATH" -TestCases @(
+        @{ ToolName = "php" }
+        @{ ToolName = "composer" }
+    ) {
+        "$ToolName --version" | Should -ReturnZeroExitCode
+    }
+
+    It "PHP Environment variables is set." {
+        Get-EnvironmentVariable "PHPROOT" | Should -Not -BeNullOrEmpty
```

Comment:
Do we want also test `Get-EnvironmentVariable "PHPROOT" | Should -Exists`?

Reply:
added this